parser-typechecker/src/Unison/Server/Endpoints/Projects.hs

```diff
@@ -0,0 +1,133 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE DeriveAnyClass #-}
+{-# LANGUAGE DeriveGeneric #-}
+{-# LANGUAGE KindSignatures #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE StandaloneDeriving #-}
+{-# LANGUAGE TypeOperators #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
```

Comment:
At a glance I don't see any orphan instances in here, did this get added by mistake or am I missing one?

It's not a huge deal if we need to add orphan instances somewhere, but in general it's nice to avoid that and I don't see any in here at  a glance. Can we remove this warning?

Reply:
You caught me in just porting over the extensions from another file. Will clean up whats not needed :D